StrawberryPieServer/src/Common.h

```diff
@@ -28,8 +28,8 @@
 #include <glm/gtx/norm.hpp>
 
 // Our includes
-#include <SharedUtils.h>
-#include <Logging.h>
+#include "SharedUtils.h"
```

Comment:
Don't do this in quotes, instead, add the right folder to the include path in the project.